sdk/containerregistry/container-registry/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
-## 1.0.0-beta.1 (Unreleased)
+## 1.0.0-beta.1 (2021-04-06)
```

Comment:
Do you need to add a line for beta.2 (Unreleased)?

Reply:
The azsdk bot will open a PR after releasing this.